pkg/plugins/resources/k8s/native/config/rbac/role.yaml

```diff
@@ -66,14 +66,6 @@ rules:
   - patch
   - update
   - watch
-- apiGroups:
-  - kuma.io
-  resources:
-  - proxytemplates/status
-  verbs:
-  - get
-  - patch
-  - update
```

Comment:
ProxyTemplate doesn't have a Status field, and it doesn't make sense for it to have a status subresource. So this change is correct, but not sure how it got in there in the first place.